pluto_build/05_qaqc.sh

```diff
@@ -10,6 +10,7 @@ import_qaqc qaqc_expected main &
 import_qaqc qaqc_aggregate main &
 import_qaqc qaqc_mismatch main &
 import_qaqc qaqc_null main &
+#import_qaqc qaqc_outlier main &
```

Comment:
Ok sorry one last thing I noticed, uncomment this line. It will fail the first time but work after that